templates/comments/list.html

```diff
@@ -141,4 +141,10 @@ <h3>{{ _('New comment') }}</h3>
             {% endif %}
         </div>
     {% endif %}
+
+    {% if comment_lock %}
+        <div style="width: 100%; padding-left: 14px" class="alert alert-warning">
+            {{ _('Comments are disabled on this page.') }}
```

Comment:
I believe if there are no comments and comments are locked, the page will display:

> There are no comments at the moment.

> Comments are disabled on this page.

Which feels fairly redundant to me. The former message should not be shown if the latter is.